### PR TITLE
[12.0][IMP] rma: incoming picking type use existing lot by default.

### PR DESCRIPTION
**Pull request by ernestotejeda**
cc @Tecnativa TT25510

### CODE REVIEW

File: rma/models/stock_warehouse.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def _get_picking_type_create_values(self, max_sequence):
                 'name': _('RMA Receipts'),
                 'code': 'incoming',
                 'use_create_lots': True,
```
Comment (pedrobaeza) [inline rma/models/stock_warehouse.py]: I think this should be false, as you don't create lots for rma

**Review by joao-p-marques — APPROVED** [review]
Functional review ok :+1:

File: rma/models/stock_warehouse.py (hunk 1 above)
Comment (chienandalu) [reply rma/models/stock_warehouse.py]: I'd say it's mostly disabled by the `use_existing_lots` setting being `True`, but maybe it's better to leave it to `False` indeed.

Comment (pedrobaeza) [reply rma/models/stock_warehouse.py]: They are not excluding in v12 AFAIK.

Comment (chienandalu) [reply rma/models/stock_warehouse.py]: It expects `use_existing_lots` to be `False` here, for example: https://github.com/odoo/odoo/blob/12.0/addons/stock/models/stock_move.py#L490

Anyway, it's not very intuitive I must say. So I agree to set it to `False` :)

**Review by chienandalu — APPROVED** [review]

**Comment by ernestotejeda** [comment]
Changes done and Conflicts resolved.
ping @pedrobaeza

**Pull request merged**